backend/lib/kjer_si/accounts.ex

```diff
@@ -76,7 +76,7 @@ defmodule KjerSi.Accounts do
   """
   def get_user_with_preload(uid, preload) do
```

Comment:
If it's id, let it be id. :)

```suggestion
  def get_user_with_preload(id, preload) do
```